pulsar/consumer_partition_test.go

```diff
@@ -92,12 +104,18 @@ func TestBatchMessageIDNoAckTracker(t *testing.T) {
 
 func TestBatchMessageIDWithAckTracker(t *testing.T) {
 	eventsCh := make(chan interface{}, 1)
+
+	tn, err := internal.ParseTopicName("topic")
+	if err != nil {
+		t.Fatal(err)
+	}
+
```

Comment:
Why do we need to parse topic name here? In the `GetTopicMetrics` method, we have called the `ParseTopicName` method, is it enough to directly handle the error value returned by that method?